amd/src/script.js

```diff
@@ -1202,6 +1208,90 @@ define(['jquery', 'core/modal_factory', 'core/modal_events'], function($, ModalF
+
+                    add_draggable_to_first_section();
+
+                    const footer = document.getElementById('page-footer');
+                    const footerIconContainer = footer.querySelector('div[data-region="footer-container-popover"]');
+
+                    var basket = document.createElement('i');
+                    basket.setAttribute('class', 'fa fa-shopping-basket');
+
+                    var basketButton = document.createElement('button');
+                    basketButton.setAttribute('class', 'btn btn-icon bg-secondary icon-no-margin btn-footer-popover');
+                    basketButton.setAttribute('style', 'z-index: 1001;');
+                    basketButton.append(basket);
+
+                    var dropAreaText = document.createElement('p');
+                    dropAreaText.setAttribute('class', 'font-weight-bold text-white');
+                    dropAreaText.innerText = str('drop_here');
+
+                    var dropArea = document.createElement('div');
+                    dropArea.setAttribute('class',
+                        'h-100 w-100 position-absolute d-flex justify-content-center align-items-center');
+                    dropArea.append(dropAreaText);
+
+                    sections.forEach(section => {
+                        drag_event_listeners(section);
+                    });
+
+                    activities.forEach(activity => {
+                        drag_event_listeners(activity);
+                    });
+
+                    /**
+                     *  Initialize events for dragging
+                     * @param {object} draggable
+                     */
+                    function drag_event_listeners(draggable) {
+                        draggable.addEventListener('dragstart', (e) => {
+                            footerIconContainer.prepend(basketButton);
+                            sharingCartBlock.children[0].classList.add('dragging_item');
+                            sharingCartBlock.append(dropArea);
+                            currentDragging = e.target;
+                        });
+
+                        draggable.addEventListener('dragend', () => {
+                            footerIconContainer.removeChild(basketButton);
+                            sharingCartBlock.children[0].classList.remove('dragging_item');
+                            sharingCartBlock.removeChild(dropArea);
+                        });
+                    }
+
+                    [basketButton, sharingCartBlock].forEach((dropzone) => {
+                        dropzone.addEventListener("dragover", (e) => {
+                            e.preventDefault();
+                        });
+                        dropzone.addEventListener("dragenter", (e) => {
+                            e.preventDefault();
+                        });
+
+                        dropzone.addEventListener("drop", () => {
+                            if (typeof currentDragging === 'object') {
```

Comment:
Is ``currentDragging`` a HTMLElement or something more specific?
If so then ``instanceof`` might be more suitable in this case.

```suggestion
                            if (currentDragging instanceof HTMLElement) {
```

Reply:
Changes added